debezium-server/debezium-server-dist/pom.xml

```diff
@@ -79,6 +79,10 @@
                     <groupId>io.debezium</groupId>
                     <artifactId>debezium-server-pulsar</artifactId>
                 </dependency>
+                <dependency>
```

Comment:
License-wise it's so we can distribute all the deps?

Reply:
Yes, as I looked it seems to be MIT. IIRC I've already went through it, just this step was forgotten in the original PR.